docs/admin/troubleshooting.md

```diff
@@ -326,112 +326,407 @@ Verify if you're using Infiniband for `fabric_iface`: in the server config. The
 
-	# Error message or timeout after dmg system query
-	$ dmg system query
-	ERROR: dmg: Unable to load Certificate Data: could not load cert: stat /etc/daos/certs/admin.crt: no such file or directory
 
-	# Workaround
+# Work around to check for daos_agent certification and start daos_agent
+	#check for /etc/daos/certs/daosCA.crt, agent.crt and agent.key
+	$ sudo systemctl enable daos_agent.service
+	$ sudo systemctl start daos_agent.service
+```
+### use daos command with invalid or wrong parameters
+```
+# Lack of providing daos pool_uuid
+$ daos pool list-cont
+pool UUID required
+rc: 2
+daos command (v1.2), libdaos 1.2.0
+usage: daos RESOURCE COMMAND [OPTIONS]
+resources:
+		  pool             pool
+		  container (cont) container
+		  filesystem (fs)  copy to and from a POSIX filesystem
+		  object (obj)     object
+		  shell            Interactive obj ctl shell for DAOS
+		  version          print command version
+		  help             print this message and exit
+use 'daos help RESOURCE' for resource specifics
+
+# Invalid sub-command cont-list
+$ daos pool cont-list --pool=$DAOS_POOL
+invalid pool command: cont-list
+error parsing command line arguments
+daos command (v1.2), libdaos 1.2.0
+usage: daos RESOURCE COMMAND [OPTIONS]
+resources:
+		  pool             pool
+		  container (cont) container
+		  filesystem (fs)  copy to and from a POSIX filesystem
+		  object (obj)     object
+		  shell            Interactive obj ctl shell for DAOS
+		  version          print command version
+		  help             print this message and exit
+use 'daos help RESOURCE' for resource specifics
+
+# Working daos pool command
+$ daos pool list-cont --pool=$DAOS_POOL
+bc4fe707-7470-4b7d-83bf-face75cc98fc
+```
+## dmg pool create failed due to no space
+```
+$ dmg pool create --size=50G mypool
+Creating DAOS pool with automatic storage allocation: 50 GB NVMe + 6.00% SCM
+ERROR: dmg: pool create failed: DER_NOSPACE(-1007): No space on storage target
+
+# Workaround: dmg storage query scan to find current available storage
+	dmg storage query usage
+	Hosts  SCM-Total SCM-Free SCM-Used NVMe-Total NVMe-Free NVMe-Used
+	-----  --------- -------- -------- ---------- --------- ---------
+	boro-8 17 GB     6.0 GB   65 %     0 B        0 B       N/A
+
+	$ dmg pool create --size=2G mypool
+	Creating DAOS pool with automatic storage allocation: 2.0 GB NVMe + 6.00% SCM
+	Pool created with 100.00% SCM/NVMe ratio
+	-----------------------------------------
+	  UUID          : b5ce2954-3f3e-4519-be04-ea298d776132
+	  Service Ranks : 0
+	  Storage Ranks : 0
+	  Total Size    : 2.0 GB
+	  SCM           : 2.0 GB (2.0 GB / rank)
+	  NVMe          : 0 B (0 B / rank)
+
+	$ dmg storage query usage
+	Hosts  SCM-Total SCM-Free SCM-Used NVMe-Total NVMe-Free NVMe-Used
+	-----  --------- -------- -------- ---------- --------- ---------
+	boro-8 17 GB     2.9 GB   83 %     0 B        0 B       N/A
+```
+### dmg pool destroy timeout
+```
```

Comment:
misleading heading: the problem is not a timeout, but the fact that you can't destroy a pool that still has a container in it without using ---force

Reply:
and i think it's dangerous to instruct people to simply use --force without first investigating what the status of that container is. you dont want to accidentally wipe out somebody's data...

Reply:
Same as above, not part of my patch. It's already part of existing document.
I will update it anyway.